coins/src/storeCoins.ts

```diff
@@ -1,31 +1,17 @@
-import { storeCoin } from "./storeCoin";
-import {
-  getCoingeckoLock,
-  releaseCoingeckoLock
-} from "../src/utils/shared/coingeckoLocks";
-import protocols from "./protocols/data";
-import { importAdapter } from "./protocols/importAdapter";
-
-const maxRetries = 4;
+import { importAdapter } from "./storeToDB/importAdapter";
+import protocols from "./storeToDB/data";
 
 async function iterateProtocols(protocolIndexes: number[]) {
   const actions = protocolIndexes
```

Comment:
is the idea here that multiple protocols will use the same adapter? if that were to happen wouldn't it be easier to store those protocols inside the adapter?